src/axom/slam/mesh_struct/IA.hpp

```diff
@@ -0,0 +1,402 @@
+// Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
```

Comment:
Here is the other main addition: IA mesh is the topological mesh data structure over a triangle or tetrahedral mesh that the Delaunay class is built on. It is implemented in terms of slam's dynamic sets, relations and maps.